web/scripts/checkout/checkoutController.js

```diff
@@ -13,7 +13,14 @@ module.exports = {
       // Spread set into array.
       const products = shoppingCart.getItems()
       const subtotal = shoppingCart.getSubtotal()
-      const stripe = Stripe('pk_test_4VicAuG5Ou9zuqA4LVIRI0dC')
+      const stripe = Stripe('pk_test_Lz1JhxYc3eAnK4P3Sc8kQHsh')
```

Comment:
I'm not sure the purpose of the key here, but is it safe to leave it here?

Reply:
👍